### ISSUE TITLE
[Feature Request]: Download link for finished items

### ISSUE THREAD

**Issue by ali-sattari** [d0u9/youtube-dl-webui#3]
User can click on the link and download finished items from the panel.

**Comment by d0u9** [comment]
Good idea! I currently use samba to export the volume videos downloaded to. However, for people who cannot access remote files via network file system, this is actually a good idea.

**Comment by raz0rsedge1975** [comment]
I would recommend this too...meanwhile you can edit the FILE LOC URL to your needs
